osquery/filesystem/windows/fileops.cpp

```diff
@@ -1211,4 +1211,10 @@ LONGLONG filetimeToUnixtime(const FILETIME& ft) {
   date.QuadPart -= adjust.QuadPart;
   return date.QuadPart / 10000000;
 }
+
+fs::path getSystemRoot() {
+  std::vector<char> winDirectory(MAX_PATH);
```

Comment:
As discussed offline, this might be better with an allocation + zeroed initialization of `MAX_PATH + 1` then providing `MAX_PATH` to the `GwtWindowsDirectory` call.